ios/FluentUI/Navigation/SideBar.swift

```diff
@@ -0,0 +1,296 @@
+                avatarViewGestureRecognizer = tapGesture;
+                view.addGestureRecognizer(tapGesture)
+            }
+
+            setupLayoutConstraints()
+        }
+    }
+
+    /// Tab bar iems to display in the top section of the side bar.
+    @objc open var topItems: [TabBarItem] = [] {
+        willSet {
+            willSetItems(in: .top)
+        }
+        didSet {
+            didSetItems(in: .top)
+        }
+    }
+
+    /// Tab bar iems to display in the bottom section of the side bar.
+    @objc open var bottomItems: [TabBarItem] = [] {
+        willSet {
+            willSetItems(in: .bottom)
+        }
+        didSet {
+            didSetItems(in: .bottom)
+        }
+    }
+
+    /// Selected tab bar item in the top section of the side bar.
+    @objc open var selectedTopItem: TabBarItem? {
+        willSet {
+            if let item = selectedTopItem {
+                itemView(with: item, in: .top)?.isSelected = false
+            }
+        }
+        didSet {
+            if let item = selectedTopItem {
+                itemView(with: item, in: .top)?.isSelected = true
+            }
+        }
+    }
+
+    @objc public override init(frame: CGRect) {
+        super.init(frame: frame)
+        initialize()
+    }
+
+    @available(*, unavailable)
+    @objc public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    /// Helper method to insert the view in a superview.
+    @objc public func insert(in superView: UIView) {
+        superView.addSubview(self)
+
+        NSLayoutConstraint.activate([
+            leadingAnchor.constraint(equalTo: superView.leadingAnchor),
+            topAnchor.constraint(equalTo: superView.topAnchor),
+            bottomAnchor.constraint(equalTo: superView.bottomAnchor)
+        ])
+    }
+
+    private enum Section: Int {
+        case top
+        case bottom
+    }
+
+    private struct Constants {
+        static let maxTabCount: Int = 5
+        static let viewWidth: CGFloat = 62.0
+        static let avatarViewSize: CGFloat = 30.0
+        static let avatarViewTopPadding: CGFloat = 18.0
+        static let topStackViewTopPadding: CGFloat = 30.0
+        static let avatarViewTopStackViewPadding: CGFloat = 34.0
+        static let bottomStackViewBottomPadding: CGFloat = 10.0
+        static let topItemSpacing: CGFloat = 38.0
+        static let bottomItemSpacing: CGFloat = 28.0
+        static let itemHeight: CGFloat = 24.0
+    }
+
+    private var layoutConstraints: [NSLayoutConstraint] = []
+    private var avatarViewGestureRecognizer: UITapGestureRecognizer?
+    private let borderLine = Separator(style: .shadow, orientation: .vertical)
+
+    private let backgroundView: UIVisualEffectView = {
+        var style = UIBlurEffect.Style.regular
+        if #available(iOS 13, *) {
+            style = .systemMaterial
```

Comment:
>systemMaterial [](start = 21, length = 14)

horizontal tab bar is systemChromeMaterial, do we want to keep that consistent?

Reply:
Yes, we do. I'll update this.